README.md

```diff
@@ -64,7 +64,7 @@ This is the primary means of running and developing the project. You may need to
 npm run serve
 ```
 
-The server automatically watches for changes to Vue files and rebuilds them. However core Rust and TypeScript file changes require a manual restart of the development server. (Please submit a PR to help it watch the `.ts` files!)
+The server automatically watches for changes to Vue files and rebuilds them. However TypeScript file changes require a manual restart of the development server. (Please submit a PR to help it watch the `.ts` files!)
```

Comment:
As it would appear, all my tests do show that TypeScript files are watched as expected. I am not sure why this was causing me problems in the past. So everything works as intended now and we can just remove this whole paragraph from the readme.